test/smoke/package/src/react-native.test.ts

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
IMO it is better to rename this file like `nativeDebug.test.ts` or something like that.